Bio/SeqIO/InsdcIO.py

```diff
@@ -901,10 +905,10 @@ def _write_contig(self, record):
             self._write_single_line("CO", text)
 
     def _write_sequence(self, record):
-        LETTERS_PER_BLOCK = 10
```

Comment:
In theory perhaps, but e.g. the ``EmblWriter`` and its subclass ``ImgtWriter`` deliberately use the same class-level constant names with different values.